projects/js-packages/critical-css-gen/src/browser-interface-playwright.ts

```diff
@@ -0,0 +1,125 @@
+import { BrowserContext, Page } from 'playwright-core';
```

Comment:
You'll probably need an optional (peer?) dependency on `playwright-core` (not `playwright`) to satisfy this for users of the library.

Or drop this whole file from the library itself, or move it into the test dir if the test using it is needed to test underlying functionality.

Reply:
> You'll probably need an optional (peer?) dependency on playwright-core (not playwright) to satisfy this for users of the library.

Apologies, I'm not sure what you mean by this.

We can't drop the file, as it's used in the cloud (since it's using the node build of this package) - https://github.com/Automattic/boost-cloud/blob/0c3a8c18063aeef0e125707932b7f9b0451e66fe/boost-hydra/src/utils/critical-css.ts#L29

Reply:
We tried creating a separate entry point here - 5c6933252b8cab6f9bd155827c329bab83757c72, but for some reason, we still ran into the missing type declarations for `playwright-core` when we tried using:

```js
import { BrowserInterfacePlaywright } from '@automattic/jetpack-critical-css-gen/playwright';
``` 

After looking at the node build, playwright is always going to be used, as that's the main entry point for that build.

Since the browser version is working (and we would be using that in Boost), we decided to move forward and update the node build in a follow up.